app/presenters/publishing_api/role_appointment_presenter.rb

```diff
@@ -15,23 +15,31 @@ def content_id
     def content
       content = BaseItemPresenter.new(
         item,
-        title: item.name,
+        title: "",
```

Comment:
I guess we ignore this and use the name of the role/person now?

Reply:
Yeah it didn't make sense to have an appointment have a title since it's never shown and is just a link between two other items that have titles.